apps/etterlatte-behandling/src/test/kotlin/behandling/tilbakekreving/TilbakekrevingDaoTest.kt

```diff
@@ -91,17 +89,26 @@ class TilbakekrevingDaoTest(val dataSource: DataSource) {
                     lagret.tilbakekreving.copy(
                         vurdering =
                             TilbakekrevingVurdering(
-                                beskrivelse = "beskrivelse",
-                                konklusjon = "konklusjon",
                                 aarsak = TilbakekrevingAarsak.ANNET,
-                                aktsomhet =
-                                    TilbakekrevingVurderingUaktsomhet(
-                                        aktsomhet = TilbakekrevingAktsomhet.GOD_TRO,
-                                        reduseringAvKravet = "Redusering av kravet",
-                                        strafferettsligVurdering = "Strafferettslig",
-                                        rentevurdering = "Rentevurdering",
-                                    ),
-                                hjemmel = TilbakekrevingHjemmel.TJUETO_FEMTEN_EN_LEDD_EN,
+                                beskrivelse = "beskrivelse",
```

Comment:
Hoi, dette var alvorleg mange felt å setje. Kanskje potensiale for forenkling etter kvart.

Reply:
Jepp, alt er optional i det skjemaet inntil videre. Skal ta en runde for å se om vi kan strukturere litt om på det der, men er det fag ønsker å ha av felter 😬